packages/browser-wallet-api-types/README.md

```diff
@@ -2,22 +2,53 @@
 
 This package includes the types for the API to be used in web applications for communicating with the Concordium browser wallet.
 
-This package changes the type of the window object, to include the injected api that is available on `window.concordium`.
+This package changes the type of the window object, to include the injected API that is available on `window.concordium`.
 
 Note that this package only contains the types and a description of the API, which is injected by the Concordium browser wallet.
 
 The actual implementation can be found [In the Concordium browser wallet repository.](https://github.com/Concordium/concordium-browser-wallet/tree/main/packages/browser-wallet-api)
 
 ## Using the API
 
-The API is automatically injected into web applications if the Concordium browser wallet extension is installed in the browser. It is made accessible by `window.concordium`. A callback (`concordiumReady`), which needs to be defined globally, is called by the injected script to notify applications that the API is ready for use. The following exemplifies how this can be done.
+The API is automatically injected into web applications if the Concordium browser wallet extension is installed in the browser. To get access to the API use the following helper function (this will be implemented in the web-sdk):
```

Comment:
```suggestion
The API is automatically injected into web applications if the Concordium browser wallet extension is installed in the browser. To get access to the API use the following helper function (this will be added to the web-sdk):
```
:thinking: